src/js/analytics.js

```diff
@@ -1,22 +1,26 @@
 import domready from './domready';
 
-export let trackEvent = (event, data) => {
-  console.log(`[Analytics disabled] Event: ${event}`); // eslint-disable-line no-console
-  console.log(data); // eslint-disable-line no-console
+export let trackEvent = (data) => {
```

Comment:
Think this change should be reverted because if GA is on this `trackEvent` is overwritten so it should be disabled at this stage